theme/login/resources/css/login.css

```diff
@@ -1,139 +1,199 @@
+  width: 70%;
+  max-width: 360px;
+  min-width: 220px;
+  margin: 0 auto;
+  padding-top: 15px;
+  padding-bottom: 5px;
+}
+
+#kc-page-title::after {
+  content: "Welcome!";
+  font-family: "Roboto-Semibold";
+  font-size: 30px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+#kc-content {
+  background-color: white;
+  margin-top: 20px;
+  margin: 0 auto;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  max-width: 360px;
+  min-width: 220px;
+  width: 70%;
+  border-radius: 16px;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  text-align: left;
+  margin-top: -2px;
 }
 
-label{
-    margin-left: 15%;
-    font-weight: bold;
+input[type="text"],
+input[type="password"],
+input[type="email"] {
+  margin-bottom: 10px;
+  width: 70%;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-color: #26c485;
+  margin-left: 15%;
+  font-size: 18px;
 }
 
-#kc-form-options{
-    margin-left: 15%;
-    margin-bottom: 10px;
-    font-size: 14px;
+input:focus {
+  outline: none;
 }
 
-#kc-form-options a{
-    color: #26c485;
+input[type="submit"] {
+  width: 70%;
+  float: center;
+  border: none;
+  border-radius: 16px;
+  box-shadow: 0 0 0;
+  font-size: 18px;
+  font-family: "Roboto-Semibold";
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background-color: #26c485;
+  color: white;
+  text-transform: uppercase;
+  margin-left: 15%;
+  margin-bottom: 15px;
 }
 
-#kc-registration{
-    margin-left: 15%;
-    font-size: 14px;
-    color: #26c485;
+label {
+  margin-left: 15%;
+  font-weight: bold;
 }
 
-#kc-registration a{
-    color: #26c485;
+#kc-form-options {
+  margin-left: 15%;
+  margin-bottom: 10px;
+  font-size: 14px;
+}
+
+#kc-form-options ~ * {
+  margin-left: 15%;
+  margin-bottom: 10px;
+  color: #26c485;
+}
+
+#kc-form-options a {
+  color: #26c485;
+}
+
+#kc-registration {
+  margin-left: 15%;
+  font-size: 14px;
+  color: #26c485;
+}
+
+#kc-registration a {
+  color: #26c485;
+}
+
+.alert {
+  margin-left: 15%;
+  margin-bottom: 5px;
+  font-size: 13px;
+}
+
+#kc-social-providers {
+  display: flex;
+  justify-content: center;
+}
+
+#kc-social-providers > ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: inline-flex;
+  justify-content: center;
+}
+
+#zocial-google {
+  font-size: 0;
+}
+
+#zocial-google::after {
+  font-size: initial;
+}
+
+#zocial-google::before {
+  content: url("../img/google_signin_buttons/web/2x/btn_google_signin_light_normal_web@2x.png");
+}
+
+#zocial-google:hover::before,
+#zocial-google:focus::before {
+  content: url("../img/google_signin_buttons/web/2x/btn_google_signin_light_focus_web@2x.png");
+}
+
+#zocial-google:active::before {
+  content: url("../img/google_signin_buttons/web/2x/btn_google_signin_light_pressed_web@2x.png");
 }
 
-.alert{
+li {
+    font-size: 14px;
+    list-style: none;
+    padding-right: 0;
+}
+
+ul {
+    padding: 0;
     margin-left: 15%;
-    margin-bottom: 5px;
-    font-size: 13px;
 }
+
+a#zocial-google {
```

Comment:
This makes no visible changes. **Please remove**

Reply:
Darfst du gerne ändern. Ich habe keine Ahnung von CSS, hab nur den Release gemacht.

Reply:
https://www.sololearn.com/Course/CSS/ - Basic knowledge.